.ado/templates/e2e-testing-uwp.yml

```diff
@@ -40,7 +40,7 @@ steps:
 
   # Wait for app to launch. A workaround to avoid WinAppDriver error: Failed to locate opened application window with appId
   - powershell: |
-      Start-Sleep -Seconds 60
+      Start-Sleep -Seconds 90
```

Comment:
Ugh, this would be more complicated but is there a way to get some sort of positive handoff?  Generally any sleep type call with a fixed number tends to lead to intermittency.  Note that I'm not asking for a fix for this review, just something to consider for the future.

Reply:
This was from RNW's CI that I copied over. We may not actually need it because of how I implemented the steps. I'm going to remove this task and observe the affects. I believe it'll be fine without it, thanks for pointing this out.